src/index.js

```diff
@@ -95,6 +105,51 @@ document.addEventListener("DOMContentLoaded", function() {
       };
     });
 
+    /* rows.map((r) => {
+      console.log(`${JSON.stringify(r.columns)}`);
+    }); */
+  
+    function addSortButton(el, idx) {
+      const img = document.createElement('img');
+      img.src = "https://uxwing.com/wp-content/themes/uxwing/download/arrow-direction/up-down-arrows-icon.png"
```

Comment:
Be better to have this locally and referenced relatively.

(Assuming no copyright/permissions issue)